docs/reference/api.rst

```diff
@@ -167,3 +167,14 @@ when bypassing the usual plugin workflow by calling internal methods directly.
 .. autoclass:: inmanta.execute.proxy.DynamicProxy()
     :members: return_value, unwrap
     :undoc-members:
+
+Rest API
+---------
+
+The rest API is also available as a `swagger spec <openapi.html#http://>`_
+
+.. automodule:: inmanta.protocol.methods
+    :members:
+
+.. automodule:: inmanta.protocol.methods_v2
+    :members:
```

Comment:
This also includes the license header in the rendered documentation. I assume that's undesired?

Reply:
I added a string before the license header to prevent that,....